bentoml/configuration/default_bentoml.yml

```diff
@@ -10,7 +10,7 @@ bento_server:
   timeout: 60
   max_request_size: 20971520
   microbatch:
-    enabled: False
+    enabled: True
```

Comment:
I suggest to just removing this option, and change this field to optional in the schema.

Reply:
Seems that we cannot simply change this field to optional (like `Optional("enabled", default=True)`) in the schema because inside [`BentoMLConfiguration.__init__()`](https://github.com/bentoml/BentoML/blob/master/bentoml/configuration/containers.py#L127), when `validate_schema=True`, [current code only validates the config dictionary](https://github.com/bentoml/BentoML/blob/master/bentoml/configuration/containers.py#L144), but does not reassign `self.config` to the validated result. That means `microbatch.enabled` is missing and this will crash the entire framework.

Solution is simply change every `SCHEMA.validate(self.config)` to `self.config = SCHEMA.validate(self.config)`. However this means when `validate_schema=False` the `microbatch.enabled` is still missing. I cannot think of a good way to solve this. Maybe we can just remove `validate_schema` argument and make the validation part mandatory. What is `validate_schema=False`'s use case?

Reply:
Since we would have "mb_max_batch_size=1" to disable the auto-batching behaviour, we may fully remove the `config.microbatch.enabled`, but keep it in the validation for backwards compatibility.

Reply:
done in https://github.com/bentoml/BentoML/pull/1652/commits/d1afffdb9183afae505714a55df1302406c57b2d